enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/discovery/HazelcastClient.java

```diff
@@ -78,6 +83,19 @@
         this.refreshPeriod = config.get( CausalClusteringSettings.cluster_topology_refresh ).toMillis();
         this.myself = myself;
         this.groups = config.get( CausalClusteringSettings.server_groups );
+        this.topologyServiceRetryStrategy = resolveStrategy();
+    }
+
+    private TopologyServiceRetryStrategy resolveStrategy()
+    {
+        Duration defaultDuration = DURATION.apply( CausalClusteringSettings.cluster_topology_refresh.getDefaultValue() );
```

Comment:
You really shouldn't need to get the default value, the config framework should handle that.